csatar-plugins/csatar/models/Scout.php

```diff
@@ -139,7 +138,7 @@ public function beforeValidate() {
             }
 
             // the registration form is required
-            $registration_form = $this->registration_form()->withDeferred($this->sessionKey)->first();
+            $registration_form = $this->registration_form()->withDeferred($this->sessionKey)->first() ?? $this->registration_form();
```

Comment:
Ez a deferred binding miatt van?

Reply:
Amiatt van, hogy az importkor csatolt .pdf fajlot deferred binding-al nem talalja meg. De lehet ki lehetne venni mostmar, hogy bevezettuk a forceSave-t.

Reply:
Erről lehet kellene beszéljünk egyet: $this->registration_form()->withDeferred($this->sessionKey)

Reply:
Azert en inkabb hagynam, mert az osszes tobbi mezonel is meg van rendesen oldva a validacio. Szoval a forceSave csak amiatt kell, hogyha rossz adatokat irnak be, akkor megis mentsuk el. Ez meg kesobb is jo lesz, amikor rendes importunk lesz, s majd kivesszuk a forceSave-t.

Reply:
Úgy emlékszem, hogy a másik PR-omban ki is vettem